atoi_conv.go

```diff
@@ -0,0 +1,14 @@
+// Implementation using Atoi
+package convert
+
+import (
+	"fmt"
+	"strconv"
+)
+
+func convAtoi(s string) (int,error) {
+	if _, err := strconv.Atoi(s); err != nil {
+		return 0, fmt.Errorf("Not correct input.")
+	}
+	return strconv.Atoi(s)
```

Comment:
Why do you call `Atoi()` function twice ? Will be better to remove error checking in 10-11 lines.